Documentation/Helm-Charts/operator-chart.gotmpl.md

```diff
@@ -11,8 +12,8 @@ This chart bootstraps a [rook-ceph-operator](https://github.com/rook/rook) deplo
 
 ## Prerequisites
 
-* Kubernetes 1.22+
-* Helm 3.x
+- Kubernetes 1.22+
```

Comment:
Let's keep the `*` for bullets. This is the standard we have across the docs, although perhaps not perfectly. We try not to switch back and forth between them, depending on the dev's editor.

Reply:
I think my formatter automatically changed this and I didn't think much of it, should be fixed now.